feathr_project/registry/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pymssql
+fastapi
+uvicorn
```

Comment:
How is this file used?

Reply:
User can run `pip install -r requirements.txt` if they want to start from the source code, also this file is used by Docker image building process.
This is standard Python working process so no doc needed, if user decided to work with the source code they should already know.